pubsub/javadsl/src/test/java/com/lightbend/lagom/javadsl/pubsub/Notification.java

```diff
@@ -4,14 +4,16 @@
 
 package com.lightbend.lagom.javadsl.pubsub;
 
-import java.io.Serializable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.lightbend.lagom.serialization.Jsonable;
 
-public class Notification implements Serializable {
+public class Notification implements Jsonable {
 
   private static final long serialVersionUID = 1L;
 
   private final String msg;
 
+  @JsonCreator
```

Comment:
I remember this being an issue for Lagom 1.5 as well, but I couldn't find any docs about it in our migration guide. Since Jackson serialization will be the default, it could be good to add some docs/reference around this.